rgd/geodata/templates/geodata/rasterentry_detail.html

```diff
@@ -16,6 +16,55 @@ <h3><span>Raster Metadata</span></h3>
   <p class="article-content">Additional Metadata: {{ object.metadata }}</p>
   <p class="article-content">Meta: {{ object.metadata }}</p>
 
-  {% include 'geodata/_includes/viewer.html' %}
+  {% include 'geodata/_includes/empty_viewer.html' %}
+
+  <script>
+    let extents = JSON.parse('{{ extents|escapejs }}');
+
+    let img = new Image();
+    img.src = extents['img_url'];
+
+    var quad = layer.createFeature('quad')
+      .data([
+        {
+          ll: {x: extents.extent.xmin, y: extents.extent.ymin },
+          ur: {x: extents.extent.xmax, y: extents.extent.ymax },
+          // image: img,
```

Comment:
This throws the error. The image URL is correct as far as I can tell (Creating an HTML element for that Image object renders the image on the page fine). GeoJs throws the error:

```
DOMException: Failed to execute 'texImage2D' on 'WebGLRenderingContext': The image element contains cross-origin data, and may not be loaded.
    at t.texture.setup (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:72:110112)
    at t.texture.bind (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:72:110473)
    at Object.<anonymous> (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:83:2166058)
    at Function.each (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:28:2956)
    at t.mapper._renderImageQuads [as render] (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:83:2166008)
    at t.renderer.render (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:72:85361)
    at t.renderWindow.render (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:72:91701)
    at t.viewer.render (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:72:102404)
    at e._renderFrame (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:39:142852)
    at H (https://cdnjs.cloudflare.com/ajax/libs/geojs/0.20.0/geo.min.js:83:181219)
```

Reply:
Fixed by:

```js
img.crossOrigin = 'anonymous'
```
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/crossOrigin